src/reducers/rootReducer.ts

```diff
@@ -0,0 +1,26 @@
+import { InitialState, Action } from '../types/default';
+import i18n from '../i18n';
+
+const initState = {
+  test: 'test',
+  theme: 'default',
+  lang: 'en',
+};
+
+const rootReducer = (state = initState, action: Action): InitialState => {
```

Comment:
Please create additional reducers within the meaning. 

appStateReduce (appState , appActions for actions )

Reply:
<img width="252" alt="Screenshot 2021-03-10 at 19 06 52" src="https://user-images.githubusercontent.com/44157749/110659594-c6e65b00-81d3-11eb-99af-8fcc45875362.png">
<img width="735" alt="Screenshot 2021-03-10 at 19 07 04" src="https://user-images.githubusercontent.com/44157749/110659632-cea5ff80-81d3-11eb-88a8-88c3109c890a.png">
<img width="428" alt="Screenshot 2021-03-10 at 19 07 42" src="https://user-images.githubusercontent.com/44157749/110659732-e5e4ed00-81d3-11eb-9efb-ac68411d7a96.png">

Reply:
something like my example for application settings